src/astro/sql/operators/agnostic_save_file.py

```diff
@@ -130,6 +129,16 @@ def file_exists(self, output_file_path, output_conn_id=None):
         except IOError:
             return False
 
+    def _set_default_values_from_dag_default(self, context):
```

Comment:
Is this code the same as the one in `load_file`? Would it be worth it for them to define this method in a single place and have both reusing it?

Reply:
@tatiana hte code is now different as load file changes the "output_table" and save file uses the "input table"